crates/neon/src/lib.rs

```diff
@@ -78,6 +78,8 @@
 //! [supported]: https://github.com/neon-bindings/neon#platform-support
 #![cfg_attr(docsrs, feature(doc_cfg))]
 
+extern crate core;
```

Comment:
What's this for?

Reply:
Oops. Junk added by my IDE that I missed. Removed.